projects/Mallard/src/helpers/date.ts

```diff
@@ -1,12 +1,20 @@
 import moment from 'moment-timezone'
-import * as RNLocalize from 'react-native-localize'
+import { languageLocale } from './locale'
+import { Platform } from 'react-native'
 
 const londonTime = (time?: string | number) => {
     if (time != null) return moment.tz(time, 'Europe/London')
     return moment.tz('Europe/London')
 }
 
-const localDate = (date: Date): string =>
-    date.toLocaleDateString(RNLocalize.getLocales()[0].languageTag)
+const localDate = (date: Date): string => {
+    if (Platform.OS === 'ios') {
+        return date.toLocaleDateString(languageLocale)
+    } else {
+        return languageLocale === 'en-US'
+            ? moment(date).format('MM/DD/YYYY')
+            : moment(date).format('DD/MM/YYYY')
```

Comment:
Looks good to me, worth adding a line here to explain why are handling locale format this way for future reference.